doc/examples/denoise_nlmeans.py

```diff
@@ -25,6 +25,8 @@
 affine = img.get_affine()
 
 mask = data[..., 0] > 80
+
+# We select only one volume for the example to run quickly.
```

Comment:
Maybe write a commented-out version of the line without this? Remember that people are reading the examples because they don't know how to do these things.